Replaced the batch sizes with 'None' instead '?'

V2dha opened the pull request:
I have replaced the default text to be shown if batch sizes are not present as 'None' instead of '?' because it is more readable and understandable during the plotting of the model. Furthermore, it avoids the confusion of any error in the model and signifies that the programmer voluntarily hasn't given any batch sizes

V2dha committed:
Replaced the batch sizes with 'None' instead '?'

I have replaced the default text to be shown if batch sizes are not present as 'None' instead of '?' because it is more readable and understandable during the plotting of the model. Furthermore, it avoids the confusion of any error in the model and signifies that the fact that the programmer voluntarily haven't given any batch sizes

V2dha commented:
@googlebot I signed it!

qlzh727 commented:
Thanks for the PR.

I think the "None" is a more standard way in python to express unassigned/unknown value. Do u have any specific example that "None" is causing any confusion?

V2dha commented:
> Thanks for the PR.
> 
> I think the "None" is a more standard way in python to express unassigned/unknown value. Do u have any specific example that "None" is causing any confusion?


There might be some confusion my pull request actually **replaces the "?" question mark with "None" because in the vis_utils.py original file the default is "?" question mark.** which is somehows signifies error rather than a missing value.

"None" specifies that voluntarily I have not specified values. But the default parameter in the keras vis_utils file is "?" question mark. Therefore I made a pull request where I have replace the "?" question mark with "None". As you stated "None" is a more standard way in python to express unassigned/unknown value rather than a question mark.
 
![Capture](https://user-images.githubusercontent.com/50369708/89230673-68d01480-d601-11ea-9abb-d35515bdca08.PNG)

Right now the default is "?" question mark if value is not specified. I changed it to "None" therefore I made a pull request. It is much easier to understand. 

For instance look at the figures below :
![model with ?](https://user-images.githubusercontent.com/50369708/89231593-450dce00-d603-11ea-8ca4-dd51b9241342.png)

To change the above image with "None"  I have to locally go to the tensorflow library installed in my computer and then change the vis_utils.py line from "?" to "None" to get the following result.

![model with "None"](https://user-images.githubusercontent.com/50369708/89231727-87370f80-d603-11ea-9336-c0da16a17baf.png)

So I think it would be easy if the default parameter for non-specified values is "None" instead of a "?" question mark that's why in this pull request **I have replaced the "?" question mark with "None" in the vis_utils.py file.**

qlzh727 reviewed (approved):
I see. Thanks for the detailed reply.
